src/Storages/ExternalStream/Kafka/KafkaSource.h

```diff
@@ -76,6 +82,24 @@ class KafkaSource final : public ISource
 
     UInt32 record_consume_batch_count = 1000;
     Int32 record_consume_timeout = 100;
+
+    /// For checkpoint
+    /// FIXME, switch to llvm-15
```

Comment:
What does this mean?

Reply:
Maybe use `std::atomic<std::shared_ptr<CheckpointContext>>`  since c++20